variables.yml

```
@@ -375,7 +375,7 @@ networks:
       max_collators: Unlimited
       active:
         rounds: 3
-        hours: 6
+        hours: 3
```

```suggestion
        hours: 6
```

Please check how is this configured